drivers/samsung/displays/mdc_protocol.cr

```diff
@@ -63,9 +63,12 @@ class Samsung::Displays::MDCProtocol < PlaceOS::Driver
     # TODO: figure out how to define indicator \xAA
     transport.tokenizer = Tokenizer.new do |io|
       bytes = io.peek
+      # disconnect if the first byte is not 0xAA
+      disconnect if bytes[0] != 170
```

Comment:
```suggestion
    transport.tokenizer = Tokenizer.new do |io|
      bytes = io.peek
      # Ensure message indicator is well-formed
      disconnect unless bytes.first == 0xAA_u8
```

You could put 0xAA_u8 into a constant to reduce instances of magic values and it even clearer

Reply:
Thanks @caspiano , yeah that is much better https://github.com/PlaceOS/drivers/pull/19/commits/cf5698e201ffe2502c3a286d4b94f88433f2a277